app/src/main/res/values/strings.xml

```diff
@@ -54,7 +65,13 @@
     <string name="copy">Copy</string>
     <string name="more">More</string>
     <string name="continue_btn">Continue</string>
-    <string name="choose_username_info">This is how others will see you on other apps.</string>
-    <string name="choose_username">Choose Username</string>
+
+    <!--IDENTITIES SCREEN-->
+    <string name="choose_an_account">Choose an Account</string>
+    <string name="to_connect_to">To connect to %1$s</string>
+
+    <!--WELCOME SCREEN-->
+    <string name="all_set">You’re all set!</string>
+    <string name="go_to">Go to Envelop %1$s</string>
```

Comment:
Envelop?

Reply:
oh I did the string format but forgot to replace the testing.... ups